### PR TITLE
refactor: update installation guide

### PR DESCRIPTION
**Pull request by castrogarciajs**
Hello @hngngn,

I have taken the trouble to improve the installation guide by adding at the end an example of how to use the CLI and how to use the generated CLI component.

Thank you for reading.

### CODE REVIEW

**Comment by hngngn** [comment]
Thank you for making the changes.  However, I noticed the manual was left untouched.  Was there a specific reason for this?

**Comment by castrogarciajs** [comment]
Oh, I thought it was not necessary to upload it to GitHub. I'm sorry, I will upload it.

**Review by hngngn — COMMENTED** [review]
What's this for?

**Comment by hngngn** [comment]
There might be a bit of a mix-up! You edited `Solid Start` and `Astro` but leave the `Manual` section untouched.

**Comment by hngngn** [comment]
> I ran `bun --cwd apps/www build:contents` as the guide said after modifying the documentation files and it gave me the above result.

This should only run when new content is added



> Sorry, I don't understand. Can you specify where the problem lies?



> There might be a bit of a mix-up! You edited `Solid Start` and `Astro` but not the `Manual` section.

There are 3 section in `Installation`. you only added examples to 2 of theme but what about the 3rd, the `Manual` one? I hope you will understand now

File: apps/www/src/contents/docs/installation/manual.mdx [hunk 1]
```diff
@@ -398,4 +398,22 @@ export const cn = (...classLists: ClassValue[]) => twMerge(clsx(classLists));
 
 You can now start adding components to your project.
 
+```bash
```
Comment (castrogarciajs) [inline apps/www/src/contents/docs/installation/manual.mdx:401]: I think this is what you were referring to, I'm sorry I didn't notice that the manual didn't have a CLI guide either, I must have thought it was already in the manual.

**Comment by castrogarciajs** [comment]
Thank you for your patience and the user's guide.

**Review by hngngn — APPROVED** [review]

**Pull request merged**